### PR TITLE
Amend email messages

### PR DESCRIPTION
**Pull request by serherca**

### CODE REVIEW

File: Book-A-Desk.Api/BookingNotifier.fs [hunk 1]
```diff
@@ -90,21 +91,31 @@ module rec BookingNotifier =
...
-        let bodyPart = TextPart("plain")
+        let bodyPart = TextPart(TextFormat.Html)
         
         bodyPart.Text <- emailNotificationDetails.Text
         mailMessage.Body <- bodyPart
         mailMessage
-        
+
+    let healthQuestionnaireMailMessage =
+            $"SVP, veuillez répondre au questionnaire sur l’état de santé. Si vous répondez oui à une des questions, nous vous demandons de rester à la maison.{Environment.NewLine}{Environment.NewLine}" +
```
Comment (steverenardbroadsign) [inline Book-A-Desk.Api/BookingNotifier.fs:101]: The sentence in French is correct but a more correct/formal way would be:

Merci de bien vouloir répondre au questionnaire sur l’état de santé. Si vous répondez oui à l'une des questions, nous vous demandons de rester chez vous.

File: Book-A-Desk.Api/BookingNotifier.fs [hunk 2]
```diff
@@ -90,21 +91,31 @@ module rec BookingNotifier =
...
             $"Opening hours: {office.OpeningHoursText}{Environment.NewLine}" +
-            "It is your responsibility to verify that the office is open for the date that you have booked"
-    
+            $"It is your responsibility to verify that the office is open for the date that you have booked.{Environment.NewLine}{Environment.NewLine}" +
+            healthQuestionnaireMailMessage
+
     let createOfficeRestriction (office : Office) =
             let (CityName officeName) = office.City
-            "We inform you that the office has some restrictions."
+            $"We inform you that the Office %s{officeName} has some restrictions.{Environment.NewLine}" +
```
Comment (steverenardbroadsign) [inline Book-A-Desk.Api/BookingNotifier.fs:116]: "the **O**ffice" => the office (to keep consistency with "the office" in the line above)

**Review by steverenardbroadsign — APPROVED** [review]
Looks fine except minor changes

File: Book-A-Desk.Api/BookingNotifier.fs (hunk 1 above)
Comment (serherca) [reply Book-A-Desk.Api/BookingNotifier.fs:101]: @steverenardbroadsign this message comes from the existing email sent right now by people in Montreal office. I don't know anything about French! XD

File: Book-A-Desk.Api/BookingNotifier.fs (hunk 2 above)
Comment (serherca) [reply Book-A-Desk.Api/BookingNotifier.fs:116]: I copied "the Office" from message sent in the confirmation. Do we need to remove uppercase in both? What do you think?

File: Book-A-Desk.Api/BookingNotifier.fs (hunk 1 above)
Comment (steverenardbroadsign) [reply Book-A-Desk.Api/BookingNotifier.fs:101]: Yes, I know but we could have asked them to modify it. It is a detail but looks so weird for me having a sentence starting with a upper case "SVP" ... :- )

Comment (serherca) [reply Book-A-Desk.Api/BookingNotifier.fs:101]: Let's set up a meeting for next week and suggest that changes to Karol ;)

**Pull request merged**